src/shared/pages/list/mpamBusinessUnits/__tests__/__snapshots__/businessAreaView.spec.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`when the businessAreaView component is mounted should render with default props 1`] = `<div />`;
```

Comment:
This looks suspiciously empty, worth double checking this test is performing as anticipated/required?

Reply:
Fixed, nice and full now